larky-api/README.md

```diff
@@ -0,0 +1,36 @@
+# Larky Versioned Engine API
+
+## How to use the API
+
+### 1) Include it as a dependency in your project.
+
+#### Maven:
+```
+<dependency>
+  <groupId>com.verygood.security</groupId>
+  <artifactId>larky-api</artifactId>
+  <version>*.*.*</version>
```

Comment:
Please change `*` to `x`, `x` is more to be a placeholder for numbered version imho